www/pfSense-pkg-squid/files/usr/local/pkg/squid_auth.xml

```diff
@@ -141,6 +141,22 @@
 			<size>5</size>
 			<default_value>5</default_value>
 		</field>
+		<field>
+			<fielddescr>Authentication Max User IP</fielddescr>
+			<fieldname>max_user_ip</fieldname>
+			<description>
+				<![CDATA[
+				This will be matched when the user attempts to log in from more than <number> different IP addresses.<br/>
```

Comment:
"Enforces a limit to the number of unique IP addresses from which a single user can login."